app/components/auth/signUp/index.tsx

```diff
@@ -13,6 +13,8 @@ import Icon from "../../../icons";
 import { OAUTH_VENDOR } from "../../../api/auth";
 import { parse } from "qs";
 
+const reactCookie = require("react-cookie");
```

Comment:
I suggest to use [store.js](https://github.com/marcuswestin/store.js/).
it's more secure and available for almost every circumstance.

Reply:
Yes sir